pulsar-client-api/src/main/java/org/apache/pulsar/client/api/ConsumerStats.java

```diff
@@ -92,7 +92,7 @@ public interface ConsumerStats extends Serializable {
     /**
      * @return Total number of messages batch receive failures
      */
-    long getTotaBatchReceivedFailed();
+    long getTotalBatchReceivedFailed();
```

Comment:
We should not change the public API here. This will break the compatibility.
You can create a new one and deprecate the old one.

Reply:
Okay, this method will not be refactor.